tabular/src/autogluon/tabular/trainer/abstract_trainer.py

```diff
@@ -600,6 +618,7 @@ def refit_single_full(self, X=None, y=None, X_val=None, y_val=None, X_unlabeled=
                 if issubclass(stacker_type, WeightedEnsembleModel):
                     # TODO: Technically we don't need to re-train the weighted ensemble, we could just copy the original and re-use the weights.
                     if self.bagged_mode:
+                        # TODO: add sample_weights here
```

Comment:
Marking as TODO to ensure this is updated. I believe this mostly fixes itself if we save/load w separately, like we do with y.

Reply:
addressed